src/base_DAO.mligo

```diff
@@ -14,48 +14,46 @@
  * Entrypoints that require no xtz to be sent.
  * This checks for such xtz condition.
  *)
-let requiring_no_xtz (param, store, config : forbid_xtz_params * storage * config)
+let requiring_no_xtz (param, store: forbid_xtz_params * storage)
     : operation list * storage =
   // check for no xtz
   if Tezos.amount > 0tez then
     (failwith forbidden_xtz : return)
   else
     match param with
-    | Drop_proposal (p)      -> drop_proposal(p, config, store)
-    | Vote (p)               -> vote(p, config, store)
-    | Flush (p)              -> flush (p, config, store)
-    | Freeze p               -> freeze(p, config, store)
-    | Unfreeze p             -> unfreeze(p, config, store)
+    | Drop_proposal (p)      -> drop_proposal(p, store)
+    | Vote (p)               -> vote(p, store)
+    | Flush (p)              -> flush (p, store)
+    | Freeze p               -> freeze(p, store)
+    | Unfreeze p             -> unfreeze(p, store)
     | Update_delegate p      -> update_delegates(p, store)
-    | Unstake_vote p         -> unstake_vote(p, config, store)
+    | Unstake_vote p         -> unstake_vote(p, store)
 
 (*
  * Entrypoints that allow xtz to be sent.
  *)
-let allowing_xtz_contract (param, store, config : allow_xtz_params_contract * storage * config) =
+let allowing_xtz_contract (param, store: allow_xtz_params_contract * storage) =
   match param with
-  | Propose (p)                -> propose(p, config, store)
+  | Propose (p)                -> propose(p, store)
   | Transfer_contract_tokens p -> transfer_contract_tokens(p, store)
   | Transfer_ownership (p)     -> transfer_ownership(p, store)
   | Accept_ownership           -> accept_ownership(store)
   | Default _                  -> (([] : operation list), store)
 
-let allowing_xtz (param, store, config : allow_xtz_params * storage * config) =
+let allowing_xtz (param, store: allow_xtz_params * storage) =
   match param with
-  | M_left p -> allowing_xtz_contract (p, store, config)
-  | M_right p -> custom_ep (p, store, config)
+  | M_left p -> allowing_xtz_contract (p, store)
+  | M_right p -> custom_ep (p, store)
 
 (*
  * The actual DAO contract, which is the same, regardless of the specific DAO
  * logic included.
  *)
-let base_DAO_contract (param, full_store : parameter * full_storage)
-    : operation list * full_storage =
-  let store : storage = full_store.0 in
-  let config : config = full_store.1 in
+let base_DAO_contract (param, full_store : parameter * storage)
```

Comment:
Why not use `store` instead of `full_store` now?

Reply:
Changed.